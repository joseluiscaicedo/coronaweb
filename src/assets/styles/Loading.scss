.containerLoading{
  height: 50vh;
}
.containerLoading__Loading {
  margin:auto;
    width: 50px;
    height: 50px;
    background-color: red;
    animation: 1s effect ease infinite;
  }

  @keyframes effect {
    0% {
      background-color: yellow;
      transform: rotate(0deg);
      border-radius:5px;
      width: 10px;
      height:10px;
    }

    100% {
      background-color: red;
      transform: rotate(360deg);
      border-radius:25px;
      width: 100px;
      height: 100px;
    }
  }