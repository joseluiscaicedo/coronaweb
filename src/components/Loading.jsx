import React from 'react';
import '../assets/styles/Loading.scss'

const Loading = () => {
  return (
    <section id='containerLoading' className='containerLoading'>
      <div id='Loading' className='containerLoading__Loading'>
      </div>
    </section>
  );
};

export default Loading;
