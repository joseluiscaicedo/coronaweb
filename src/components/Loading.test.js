import React from 'react';
import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/react'
import Loading from './Loading';


describe('validate Loading Component', () =>{
  test('renders content', () => {
    const component = render(<Loading />)

    expect(component.length)>0;

  });
  test('Generates Snapshot for Loading Component', () => {
    const { container } = render(
      <Loading />
    );
    expect(container.firstChild).toMatchSnapshot();
  });

})