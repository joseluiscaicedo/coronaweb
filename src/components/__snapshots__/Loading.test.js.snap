// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validate Loading Component Generates Snapshot for Loading Component 1`] = `
<section
  class="containerLoading"
  id="containerLoading"
>
  <div
    class="containerLoading__Loading"
    id="Loading"
  />
</section>
`;
